scripts/aws/cfn/microservice-dev-cfg.json

```diff
@@ -28,10 +28,10 @@
     "PnSsUriBuilderGetFileWithPatchConfiguration": "CLIENT_SPECIFIC",
     "JavaToolOptions": "-XX:MaxDirectMemorySize=128M",
     "GestoreBucketGetObjectChunkSize": "10485760",
-    "PnSignProviderSwitch": "1999-02-01T10:00:00Z;aruba",
+    "PnSignProviderSwitch": "1970-01-01T00:00:00Z;namirial",
```

Comment:
lo cambi in modo che fino a ieri è impostato Aruba e dalle 23 di stanotte Namirial?

Reply:
modificato

Reply:
non l'hai modificato

Reply:
ora è corretto